plasmapy/particles/tests/test_particle_collections.py

```diff
@@ -322,3 +323,43 @@ def test_particle_multiplication(method, particle):
     """
     particle_list = getattr(particle, method)(3)
     assert particle_list == [particle, particle, particle]
+
+
+def test_particle_list_with_no_arguments():
+    """Test that `ParticleList()` returns an empty `ParticleList`."""
+    empty_particle_list = ParticleList()
+    assert isinstance(empty_particle_list, ParticleList)
+    assert len(empty_particle_list) == 0
+
+
+def test_ion_list_example():
+    ions = ionic_levels("He-4")
+    assert np.allclose(ions.charge_number, [0, 1, 2])
```

Comment:
```suggestion
    np.testing.assert_equal(ions.charge_number, [0, 1, 2])
```

This is a bit better on the error display, and actually equal is completely fine for ints